blob/azureblob/azureblob.go

```diff
@@ -561,7 +560,7 @@ func (b *bucket) Copy(ctx context.Context, dstKey, srcKey string, opts *driver.C
 			return err
 		}
 	}
-	resp, err := dstBlobURL.StartCopyFromURL(ctx, srcURL, md, mac, bac, at, btm)
+	resp, err := dstBlobURL.StartCopyFromURL(ctx, srcURL, md, mac, bac, at, nil /* blogsTagMap */)
```

Comment:
nit: BlobTagsMap (typo)

Reply:
Haha, not sure how I mistyped that. Guess I must be thinking about blogs...